packages/lab/src/radio-button/RadioButtonIcon.css

```diff
@@ -1,78 +1,86 @@
-/* Styles applied to root component */
+/* Styles applied to radio button icon */
 .saltRadioButtonIcon {
-  --radioButton-icon-fill: transparent;
-  --radioButton-icon-fill-checked: var(--salt-selectable-borderColor-selected);
-  --radioButton-icon-fill-checked-disabled: var(--salt-selectable-borderColor-selectedDisabled);
-  --radioButton-icon-fill-error: var(--salt-status-error-borderColor);
-  --radioButton-icon-fill-error-disabled: var(--salt-selectable-borderColor-selectedDisabled);
-
-  width: var(--saltRadioButton-icon-svg-size, var(--salt-size-selectable));
-  height: var(--saltRadioButton-icon-svg-size, var(--salt-size-selectable));
+  width: var(--salt-size-selectable);
+  height: var(--salt-size-selectable);
   border-radius: 50%;
   flex: none;
   margin-top: var(--salt-size-border);
 }
 
-/* Styles applied to RadioButtonIcon border */
-.saltRadioButtonIcon-border {
-  fill: var(--saltRadioButton-icon-fill, var(--radioButton-icon-fill));
+/* Styles applied to radio button icon circle */
+.saltRadioButtonIcon-circle {
+  fill: var(--saltRadioButton-icon-fill, var(--salt-container-primary-background));
   stroke: var(--saltRadioButton-icon-strokeColor, var(--salt-selectable-borderColor));
```

Comment:
just spoke with @navkaur76 and i think this one is supposed to be selectable-foreground, and same for wherever border is used (i noted in the specs)
she needs to clarify with @pseys since naturally i would assume border is the correct token here too

Reply:
the actual colours don't seem to match the design though

Reply:
agreed, when i inspected it does use border which is why i msged nav